ingq/illust/exception/illust_exception.py

```
@@ -0,0 +1,12 @@
+from fastapi import status
+
+from core.exception.custom_exception import BusinessException
+
+
+class IllustDoesNotBelongToBookException(BusinessException):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)